test/e_chronicler_web/controllers/journal_entry_controller_test.exs

```diff
@@ -17,5 +17,12 @@ defmodule EChroniclerWeb.JournalEntryControllerTest do
     assert html_response(conn, 200) =~ "Supercilious"
   end
 
+  test "Single Journal Entry Is Displayed At /:id", %{conn: conn} do
+    {:ok, journal_entry} = EChronicler.Models.JournalEntry.create_journal_entry(%{author: "Bob", title: "Hello", entry: "Test."})
+    {:ok, _journal_entry_irrelevant} = EChronicler.Models.JournalEntry.create_journal_entry(%{author: "Betty", title: "Goodbye", entry: "Test."})
+    conn = get(conn, "/#{journal_entry.id}")
+    assert html_response(conn, 200) =~ "Hello"
+    assert !String.contains?(html_response(conn, 200), "Goodbye")
```

Comment:
Your tests will probably be less brittle if you assert directly on the properties of the structs, e.g. 

```elixir
assert html_response(conn, 200) =~ journal_entry.title
refute html_response(conn, 200) =~ other_journal_entry.title
```

Reply:
Approved from me so long as Hana's comments are met!  I steered Grace towards the `String.contains`...and I'm very glad now to learn about `refute`!